### PR TITLE
Updated og image for docs

### PR DESCRIPTION
**Pull request by hqasmei**
## Description
- Added generateOgImages, og-templates, and index.png.ts to generate the og images dynamically
- Had to rearrange and remove previous files used to first try to get the og image to work

## Related Issue
Related to the #94 issue

## Motivation and Context
To make it more efficient to the user to know what the link is about, in this case, specifically pertaining to the docs.

## How Has This Been Tested?
Tested locally by adding, .png at the end of the url and seeing if the related OG Image shows up. 

## Screenshots (if appropriate):
introduction

installation

### CODE REVIEW

File: docs/src/components/Seo/HeadSEO.astro [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
-
+import { SITE_URL } from "@/lib/utils";
+ 
 type Props = { canonicalUrl: URL } & CollectionEntry<"docs">["data"];
 
 const { ogLocale, image, title, description, canonicalUrl } = Astro.props;
 const formattedContentTitle = `${title} | ${SITE.title}`;
-const imageSrc = image?.src ?? OPEN_GRAPH.image.src;
-const canonicalImageSrc = new URL(imageSrc, Astro.site);
+// const imageSrc = image?.src ?? OPEN_GRAPH.image.src;
```
Comment (danstepanov) [inline docs/src/components/Seo/HeadSEO.astro:10]: is this still necessary?

**Comment by danstepanov** [comment]
@hqasmei can you walk me through this PR? There seem to be a lot of changes for an OG image fix?

File: docs/src/components/Seo/HeadSEO.astro (hunk 1 above)
Comment (hqasmei) [reply docs/src/components/Seo/HeadSEO.astro]: Nope, I'll update it.

**Comment by danstepanov** [comment]
@hqasmei just an FYI that when this is ready, you'll need to click "Ready for review"

File: docs/src/consts.ts [hunk 2]
```diff
@@ -53,6 +53,9 @@ export const SIDEBAR: Sidebar = {
       { text: "Introduction", link: "en/introduction" },
       { text: "Installation", link: "en/installation" },
     ],
+    // Usage: [
```
Comment (danstepanov) [inline docs/src/consts.ts:56]: @hqasmei is this needed?

Comment (hqasmei) [reply docs/src/consts.ts]: Had it there as reference to test the og stuff. Will removed it.

**Pull request merged**